tests/ui/README.md

```diff
@@ -0,0 +1,68 @@
+## CodaLab UI Tests
+
+We test the CodaLab UI by using [Selenium](https://pypi.org/project/selenium/#history) and 
+[diffimg](https://pypi.org/project/diffimg/#history) for our screenshots comparison tool. Currently, our front end 
+automation only runs in Chrome and Firefox. We hope to support more browsers in the future. 
+
+### Setting up the tests on your local machine
+
+#### One-time setup
+
+1. If you haven't already, install the Python dependencies specified in `requirements.txt`, located in the root of the 
+repository, by running `pip install -r requirements.txt`.
+2. Next, install [ChromeDriver](https://chromedriver.chromium.org/getting-started) (for Chrome) and 
+[GeckoDriver](https://github.com/mozilla/geckodriver) (for Firefox). These drivers are the links between the Selenium 
+tests and their respective browsers.
+    1. Run `./scripts/test-setup.sh` and place the installed GeckoDriver on your path by running 
+    `export PATH=$PATH:/usr/bin/geckodriver`. This is not needed for ChromeDriver.
+    2. An alternative is to simply use [Homebrew](https://brew.sh/) and run `brew install geckodriver && brew cask 
+    install chromedriver`. 
+
+#### Running the tests locally
+
+1. From the root of the repository, start the CodaLab service by running `./codalab_service.py start -bd`.
+2. Populate your local machine with a comprehensive sample worksheet by running 
+`docker exec -it codalab_rest-server_1 /bin/bash -c "python3 scripts/create_sample_worksheet.py"`. 
+3. Start the UI tests by running `python tests/ui/ui_tester.py`. You can also pass in an additional argument 
```

Comment:
I wasn't able to get this working locally on my computer (got issue - "Message: Unable to find a matching set of capabilities") but will need to debug that further. It works on travis anyway, which should be enough.

Reply:
It's important that people set these test up on their development environment as well. Did `test-setup.sh` run successfully for you? Can you check if ChromeDriver and GeckoDriver are installed?

Reply:
I tried again locally and it seems to work for me. I can merge in the code and have others try it out. Based on feedback, I can update the documentation on a follow-up PR, where I need to add more tests anyways.